plugins/receivers/lark/config.go

```diff
@@ -0,0 +1,105 @@
+package lark
+
+import (
+	"fmt"
+
+	"github.com/goto/siren/pkg/errors"
+	"github.com/goto/siren/pkg/httpclient"
+	"github.com/goto/siren/pkg/retry"
+)
+
+// AppConfig is a config loaded when siren is started
+type AppConfig struct {
+	APIHost    string            `mapstructure:"api_host"`
+	Retry      retry.Config      `mapstructure:"retry"`
+	HTTPClient httpclient.Config `mapstructure:"http_client"`
+}
+
+// LarkCredentialConfig is config that needs to be passed when a new lark
+// receiver is being added
+type LarkCredentialConfig struct {
+	ClientID     string `mapstructure:"client_id"`
+	ClientSecret string `mapstructure:"client_secret"`
+	AuthCode     string `mapstructure:"auth_code"`
```

Comment:
What is `auth_code` for lark? Is it being used?

Reply:
removed